.github/workflows/pr-name-check.yaml

```diff
@@ -0,0 +1,28 @@
+name: PR Name Check
+
+on:
+  pull_request:
+    branches:
+      - 'dev'
+    types: [opened, edited, synchronize]
+
+jobs:
+  check-pr-name:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
```

Comment:
Can we change this to v4? I believe v2 uses a deprecated version of node which posts warnings that will soon become errors in GitHub.